terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,62 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
```

Comment:
Terraform documentation suggests to commit this file. See https://stackoverflow.com/questions/67963719/should-terraform-lock-hcl-be-included-in-the-gitignore-file